aggregator/src/aggregator/test_util.rs

```diff
@@ -200,3 +203,15 @@ pub async fn assert_task_aggregation_counter(
         sleep(Duration::from_millis(100)).await;
     }
 }
+
+/// Extension methods on [`Clock`].
+pub trait ClockExt: Sized {
```

Comment:
The reason we do extension traits on other time related types has to do with the layering of the crates. In particular, `janus_messages` gets used by Firefox's DAP client (I think; or at least we want that to be possible) so we want it to have minimal dependencies, and for that reason it doesn't pull in `chrono`. Instead we do that in `janus_core` and up.

However it seems like your objective is to have some functionality on `Clock`s that's only available in test configurations. Sticking this in `test_util` achieves that, but it could be simpler to stick this method in `janus_core::time::Clock` with `#[cfg(test)]` or gate it on feature `test_util`. Then at least all the clock logic is in one place. However I don't object to how you've done it here.

Reply:
I think 23805016 works!